### ISSUE TITLE
Add PHI codes to 'spring-fat' integration test

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/opeo-maven-plugin#226]
We need to add PHI printing to the 'spring-fat' integration test. We can print PHI expressions from EO files using the `xmir-to-phi` goal of the `eo-maven-plugin`:

```xml
<plugin>
  <groupId>org.eolang</groupId>
  <artifactId>eo-maven-plugin</artifactId>
  <version>0.36.0</version>
  <executions>
    <execution>
      <id>convert-xmir-to-phi</id>
      <phase>process-classes</phase>
      <goals>
        <goal>xmir-to-phi</goal>
      </goals>
      <configuration>
        <phiInputDir>${project.build.directory}/generated-sources/jeo-xmir</phiInputDir>
        <phiOutputDir>${project.build.directory}/generated-sources/jeo-phi</phiOutputDir>
      </configuration>
    </execution>
  </executions>
</plugin>
```

**Comment by 0pdd** [comment]
@volodya-lombrozo 3 puzzles [#245](https://github.com/objectionary/opeo-maven-plugin/issues/245), [#246](https://github.com/objectionary/opeo-maven-plugin/issues/246), [#247](https://github.com/objectionary/opeo-maven-plugin/issues/247) are still not solved.

**Comment by 0pdd** [comment]
@volodya-lombrozo 5 puzzles [#245](https://github.com/objectionary/opeo-maven-plugin/issues/245), [#246](https://github.com/objectionary/opeo-maven-plugin/issues/246), [#247](https://github.com/objectionary/opeo-maven-plugin/issues/247), [#255](https://github.com/objectionary/opeo-maven-plugin/issues/255), [#256](https://github.com/objectionary/opeo-maven-plugin/issues/256) are still not solved.

**Comment by 0pdd** [comment]
@volodya-lombrozo 6 puzzles [#245](https://github.com/objectionary/opeo-maven-plugin/issues/245), [#246](https://github.com/objectionary/opeo-maven-plugin/issues/246), [#247](https://github.com/objectionary/opeo-maven-plugin/issues/247), [#255](https://github.com/objectionary/opeo-maven-plugin/issues/255), [#256](https://github.com/objectionary/opeo-maven-plugin/issues/256), [#268](https://github.com/objectionary/opeo-maven-plugin/issues/268) are still not solved.

**Comment by 0pdd** [comment]
@volodya-lombrozo 5 puzzles [#245](https://github.com/objectionary/opeo-maven-plugin/issues/245), [#246](https://github.com/objectionary/opeo-maven-plugin/issues/246), [#247](https://github.com/objectionary/opeo-maven-plugin/issues/247), [#255](https://github.com/objectionary/opeo-maven-plugin/issues/255), [#268](https://github.com/objectionary/opeo-maven-plugin/issues/268) are still not solved; solved: [#256](https://github.com/objectionary/opeo-maven-plugin/issues/256).

**Comment by 0pdd** [comment]
@volodya-lombrozo 4 puzzles [#245](https://github.com/objectionary/opeo-maven-plugin/issues/245), [#246](https://github.com/objectionary/opeo-maven-plugin/issues/246), [#247](https://github.com/objectionary/opeo-maven-plugin/issues/247), [#255](https://github.com/objectionary/opeo-maven-plugin/issues/255) are still not solved; solved: [#256](https://github.com/objectionary/opeo-maven-plugin/issues/256), [#268](https://github.com/objectionary/opeo-maven-plugin/issues/268).

**Comment by 0pdd** [comment]
@volodya-lombrozo 2 puzzles [#245](https://github.com/objectionary/opeo-maven-plugin/issues/245), [#246](https://github.com/objectionary/opeo-maven-plugin/issues/246) are still not solved; solved: [#247](https://github.com/objectionary/opeo-maven-plugin/issues/247), [#255](https://github.com/objectionary/opeo-maven-plugin/issues/255), [#256](https://github.com/objectionary/opeo-maven-plugin/issues/256), [#268](https://github.com/objectionary/opeo-maven-plugin/issues/268).
